### PR TITLE
fix: Convert house gumps to static/dynamic gumps

### PR DESCRIPTION
**Pull request by IcculusC**
clean up and organize several house related gumps, but not the primary house management gumps (yet)

### CODE REVIEW

File: Projects/UOContent/Gumps/Houses/SetSecureLevelGump.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            builder.AddHtmlLocalized(45, 130, 150, 20, 1063455, GetColor(SecureLevel.Guild)); // Guild Members
+        }
+
+        builder.AddButton(10, 130 + offset, GetFirstID(SecureLevel.Anyone), 4007, 4);
+        builder.AddHtmlLocalized(45, 130 + offset, 150, 20, 1061626, GetColor(SecureLevel.Anyone)); // Anyone
+    }
+
+    public int GetColor(SecureLevel level) => _info.Level == level ? 0x7F18 : 0x7FFF;
+
+    public int GetFirstID(SecureLevel level) => _info.Level == level ? 4006 : 4005;
```
Comment (kamronbatman) [inline Projects/UOContent/Gumps/Houses/SetSecureLevelGump.cs:66]: Do these need to be public? If not, let's make them private.
Also add `[MethodImpl(MethodImplOptions.AggressiveInlining)]` to both.

Comment (IcculusC) [reply Projects/UOContent/Gumps/Houses/SetSecureLevelGump.cs:66]: Kam I have no idea 🤣 but ok

Comment (IcculusC) [reply Projects/UOContent/Gumps/Houses/SetSecureLevelGump.cs]: made `private` and added inlining

File: Projects/UOContent/Gumps/Houses/AcceptHouseTransferGump.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+        builder.AddImageTiled(10, 10, 400, 20, 2624);
+        builder.AddAlphaRegion(10, 10, 400, 20);
+
+        builder.AddHtmlLocalized(10, 10, 400, 20, 1060635, 30720); // <CENTER>WARNING</CENTER>
+
+        builder.AddImageTiled(10, 40, 400, 200, 2624);
+        builder.AddAlphaRegion(10, 40, 400, 200);
+
+        /* Another player is attempting to initiate a house trade with you.
```
Comment (kamronbatman) [inline Projects/UOContent/Gumps/Houses/AcceptHouseTransferGump.cs:33]:
```suggestion
        /*
          * Another player is attempting to initiate a house trade with you.
```

File: Projects/UOContent/Gumps/Houses/ConfirmDemolishHouseGump.cs [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+        builder.AddImageTiled(10, 10, 400, 20, 2624);
+        builder.AddAlphaRegion(10, 10, 400, 20);
+
+        builder.AddHtmlLocalized(10, 10, 400, 20, 1060635, 30720); // <CENTER>WARNING</CENTER>
+
+        builder.AddImageTiled(10, 40, 400, 200, 2624);
+        builder.AddAlphaRegion(10, 40, 400, 200);
+
+        /* You are about to demolish your house.
```
Comment (kamronbatman) [inline Projects/UOContent/Gumps/Houses/ConfirmDemolishHouseGump.cs:33]:
```suggestion
        /*
         * You are about to demolish your house.
```

File: Projects/UOContent/Gumps/Houses/ConfirmDemolishHouseGump.cs [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    }
+
+    public override void OnResponse(NetState state, in RelayInfo info)
+    {
+        if (info.ButtonID != 1 || _house.Deleted)
+        {
+            return;
+        }
+
+        if (_house.IsOwner(state.Mobile))
```
Comment (kamronbatman) [inline Projects/UOContent/Gumps/Houses/ConfirmDemolishHouseGump.cs:59]: Invert the if statement with a return to reduce nesting.

File: Projects/UOContent/Gumps/Houses/ConfirmDemolishHouseGump.cs [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+                // You cannot do that that while you still have unclaimed contract vendor inventory in your house.
+                state.Mobile.SendLocalizedMessage(1062681);
+                return;
+            }
+
+            if (state.Mobile.AccessLevel > AccessLevel.Player)
+            {
+                state.Mobile.SendMessage("You do not get a refund for your house as you are not a player");
+                _house.RemoveKeys(state.Mobile);
+                _house.Delete();
```
Comment (kamronbatman) [inline Projects/UOContent/Gumps/Houses/ConfirmDemolishHouseGump.cs:102]: Once the if statement above is inverted, this is redundant.

File: Projects/UOContent/Gumps/Houses/SetSecureLevelGump.cs [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+        builder.AddHtmlLocalized(45, 90, 150, 20, 1061278, GetColor(SecureLevel.CoOwners)); // Co-Owners
+
+        builder.AddButton(10, 110, GetFirstID(SecureLevel.Friends), 4007, 3);
+        builder.AddHtmlLocalized(45, 110, 150, 20, 1061279, GetColor(SecureLevel.Friends)); // Friends
+
+        var houseOwner = _house.Owner;
+        if (Guild.NewGuildSystem && houseOwner?.Guild != null &&
+            ((Guild)houseOwner.Guild).Leader == houseOwner
+           ) // Only the actual House owner AND guild master can set guild secures
+        {
```
Comment (kamronbatman) [inline Projects/UOContent/Gumps/Houses/SetSecureLevelGump.cs:56]:
```suggestion
        // Only the actual House owner AND guild master can set guild secures
        if (Guild.NewGuildSystem && houseOwner.Guild is Guild guild && guild.Leader == houseOwner)
        {
```

File: Projects/UOContent/Gumps/Houses/AcceptHouseTransferGump.cs (hunk 2 above)
Comment (IcculusC) [reply Projects/UOContent/Gumps/Houses/AcceptHouseTransferGump.cs:33]: thought I did this

**Pull request merged**